app.lua

```diff
@@ -67,9 +67,14 @@ require 'responses'
 
 -- Make cookies persistent
 app.cookie_attributes = function(self)
+    local secure = ' Secure;'
+    -- Exclude security restriction in development
+    if package.loaded.config._name == 'development' then
```

Comment:
to make this more portable it would be better if instead of checking for the environment name `'development'` you instead checked a config property called something like `use_secure_cookies`

Reply:
this would also allow easily toggling this value even when in development

Reply:
oh yeah, makes sense